app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -11,12 +11,16 @@ angular.module('topologyApp', ['kubernetesUI'])
         else {
             id = '/'+ (/container_topology\/show\/(\d+)/.exec($location.absUrl())[1]);
         }
-
+        currentSelectedKinds = $scope.kinds;
```

Comment:
Please don't use global variables like this...  If it indeed needs to be a global, add it under the ManageIQ object in miq_global.js and document it there. But maybe using an angular service to store it would make more sense?